pkgs/by-name/ra/rattler-build/package.nix

```
@@ -0,0 +1,54 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  versionCheckHook,
+
+  openssl,
+  pkg-config,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "rattler-build";
+  version = "0.29.0";
+
+  src = fetchFromGitHub {
+    owner = "prefix-dev";
+    repo = pname;
```

```suggestion
    repo = "rattler-build";
```
templating with `pname` is usually discouraged.